src/lib.rs

```diff
@@ -1,37 +1,304 @@
+//!  external tooling like [`wasm-pack`](https://github.com/rustwasm/wasm-pack).
+//!
+//! # Minimum Supported Rust Version
+//!
+//! This crate requires **Rust 1.58.1** at a minimum.
+//!
+//! # Setup
+//!
+//! The best way to add xtask-wasm to your project is to create a workspace
+//! with two packages: your project's package and the xtask package.
+//!
+//! ## Create or update a project using xtask
+//!
+//! * New project - You can create a new project using xtask, you can use the
+//!     following:
+//!     ```console
+//!     mkdir project_name
+//!     cd project_name
+//!     cargo new project
+//!     cargo new xtask
+//!     touch Cargo.toml
+//!     ```
+//!     Open the workspace's Cargo.toml and add the following:
+//!     ```toml
+//!     [workspace]
+//!     members = [
+//!         "project",
+//!         "xtask",
+//!     ]
+//!     ```
+//!
+//! * Project with a single package - If you project contains only one package,
+//!     move all the content of the project expect the `.git` directory into a
+//!     new directory named after the package name at the root of the project.
+//!     * Create a new package for the xtasks using the following:
+//!         ```console
+//!         cargo new --bin xtask
+//!         ```
+//!     * Create a new Cargo.toml at the root of the project and add the following:
+//!         ```toml
+//!         [workspace]
+//!         members = [
+//!             "project",
+//!             "xtask",
+//!         ]
+//!         ```
+//!         Replace `project` by the name of the project package
+//!
+//! * Project with a workspace - If your project already use a workspace,
+//!     * Create a new package:
+//!         ```console
+//!         cargo new --bin xtask
+//!         ```
+//!     * Add the new package to your workspace's Cargo.toml with the
+//!         workspace members field
+//!
+//! ## Add a command alias
+//!
+//! Create a `.cargo` directory at the workspace root and add a file named
+//! `config.toml` with the following content:
+//!
+//! ```toml
+//! [alias]
+//! xtask = "run --package xtask --"
+//! ```
+//!
+//! Now you can run your xtask package using:
+//!
+//! ```console
+//! cargo xtask
+//! ```
+//!
+//! ## Directory layout example
+//!
+//! If the name of the project package is `app`, the directory layout should
+//! look like this:
+//!
+//! ```bash
+//! project
+//! ├── .cargo
+//! │   └── config.toml
+//! ├── Cargo.toml
+//! ├── app
+//! │   ├── Cargo.toml
+//! │   └── src
+//! │       └── ...
+//! └── xtask
+//!     ├── Cargo.toml
+//!     └── src
+//!         └── main.rs
+//! ```
+//!
+//! You can find more informations about xtask
+//! [here](https://github.com/matklad/cargo-xtask/).
+//!
+//! ## Use xtask-wasm as a dependency
+//!
+//! Finally, add the following to the xtask package's Cargo.toml:
+//!
+//! ```toml
+//! [dependencies]
+//! xtask-wasm = "0.1.0"
+//! ```
+//!
+//! # Usage
+//!
+//! This library give you 3 types:
```

Comment:
Watch your conjugation. You seem to repeat the same mistakes over and over again
```suggestion
//! This library gives you 3 [clap](link-to-clap) structs:
```